src/Components/Address/AddressAutocompleteInput.tsx

```diff
@@ -345,3 +346,58 @@ export const AddressAutocompleteInput = ({
     />
   )
 }
+
+const fetchSuggestionsWithThrottle = throttle(
```

Comment:
Moved a lot of verbose logic out of the component definition. This was previously memoized in the component because of the usage of throttle.